kudo-operator/docs/latest/submission.md

```diff
@@ -1,2 +1,160 @@
 Submitting Spark Applications
----
\ No newline at end of file
+---
+
+In order to deploy your application to Kubernetes using Kudo Spark Operator, first you need to describe it as a Kubernetes object. To do that, you need to create a specification in `yaml` format, that will contain all the configuration required for the application.
+
+Let's take a simple `SparkPi` application as an example. The `yaml` specification could be found here: [spark-pi.yaml](resources/spark-pi.yaml)
+
+```yaml
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: spark-pi
+  namespace: spark
+spec:
+  type: Scala
+  mode: cluster
+  image: "mesosphere/spark:spark-2.4.3-hadoop-2.9-k8s"
+  imagePullPolicy: Always
+  mainClass: org.apache.spark.examples.SparkPi
+  mainApplicationFile: "local:///opt/spark/examples/jars/spark-examples_2.11-2.4.3.jar"
+  arguments:
+    - "150000"
+  sparkConf:
+    "spark.ui.port": "4041"
+  sparkVersion: "2.4.3"
+  restartPolicy:
+    type: Never
+  driver:
+    cores: 1
+    memory: "512m"
+    labels:
+      version: 2.4.3
+    serviceAccount: spark-driver
+  executor:
+    cores: 1
+    instances: 2
+    memory: "512m"
+    labels:
+      version: 2.4.3
+```
+
+Basically, all the Spark application configuration is placed under `spec` section. Here you can specify Spark related configuration properties, such as number of executors, number of cores for drivers/executors, amount of memory and the other. There is also a `sparkConf` section, where you can place configuration parameters in a form of key-value pairs. In the example we override the default `spark.ui.port` with a custom value.
+
+After the application spec is ready, the following command can be used to run it on the operator:
+```
+$ kubectl create -f ./resources/spark-pi.yaml
+```
+
```

Comment:
it makes sense to add a `list` command (`k get  sparkapplications -n spark`) to display all Spark Applications and verify that `spark-pi` is created before getting the specific one